README.md

```diff
@@ -545,6 +545,88 @@ export const Component = () => {
 };
 ```
 
+### Redirects
```

Comment:
We carefully design and write our README (which will be on the website).
Can you write it down in docs instead for now?

Reply:
Sure! There you go.